### PR TITLE
HDDS-4623. Update CountQuota to NamesapceQuota

### PR DESCRIPTION
**Pull request by amaliujia**
## What changes were proposed in this pull request?

Use more description words "namespace quota" to replace "count quota".

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-4623

## How was this patch tested?

UT

### CODE REVIEW

**Comment by amaliujia** [comment]
R: @captainzmc

File: hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/OzoneVolume.java [hunk 1]
```diff
@@ -300,7 +300,7 @@ public void clearSpaceQuota() throws IOException {
    *
    * @throws IOException
    */
-  public void clearCountQuota() throws IOException {
+  public void clearNamespaceQuota() throws IOException {
```
Comment (captainzmc) [inline hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/OzoneVolume.java:303]: Same as above

File: hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/OzoneBucket.java [hunk 2]
```diff
@@ -432,7 +432,7 @@ public void clearSpaceQuota() throws IOException {
    *
    * @throws IOException
    */
-  public void clearCountQuota() throws IOException {
+  public void clearNamespaceQuota() throws IOException {
```
Comment (captainzmc) [inline hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/OzoneBucket.java:435]: The method description also needs to be update.

File: hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/ClearSpaceQuotaOptions.java [hunk 3]
```diff
@@ -30,14 +30,14 @@
 
   @CommandLine.Option(names = {"--namespace-quota"},
       description = "clear count quota")
```
Comment (captainzmc) [inline hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/ClearSpaceQuotaOptions.java:32]: update description.

**Review by captainzmc — COMMENTED** [review]
Thanks for the update. LGTM overall, just some minor comments. You can also check if there are any other modifications needed.

**Comment by amaliujia** [comment]
@captainzmc  

comments addressed.

**Review by ayushtkn — APPROVED** [review]
I tried to find any other usage of count quota, but couldn't find any.
Changes LGTM

**Review by captainzmc — APPROVED** [review]
LGTM +1. Thanks for @amaliujia's patch and @ayushtkn's review.

**Pull request merged**